tox.ini

```diff
@@ -17,7 +17,7 @@ commands=pidiff pubtools-ami .
 [testenv:cov]
 usedevelop=true
 commands=
-	pytest --cov-report=html --cov-report=xml --cov=pubtools {posargs}
+	pytest --cov-report=html --cov-report=xml --cov=src/pubtools/_ami {posargs}
```

Comment:
minor: let's be consistent with the convention like "--cov=pubtools._ami" as used elsewhere